tools/shell/shell_runner.cpp

```diff
@@ -37,11 +43,23 @@ int main(int argc, char* argv[]) {
     if (readOnlyMode) {
         systemConfig.readOnly = true;
     }
+    if (!pathToHistory.empty() && pathToHistory.back() != '/') {
+        pathToHistory += '/';
+    }
+    pathToHistory += "history.txt";
+    std::unique_ptr<VirtualFileSystem> vfs = std::make_unique<VirtualFileSystem>();
+    try {
+        std::unique_ptr<FileInfo> fp = vfs->openFile(pathToHistory, O_CREAT);
+    }
+    catch (std::exception& e) {
```

Comment:
Kuzu filesystem uses customized exception handling mechanism instead of std::exception.
So catch (common::Exception& e) here.